workshops.md

```diff
@@ -1,91 +1,175 @@
     address or a software-carpentry.org email address, whichever the
-    organization prefers.  All conversations must be recorded in a
-    shared central location approved by the Steering Committee (so
-    that we don't have to dig around in people's private mail archives
-    two years from now for conversations).
-
-9.  All records of individuals' participation in workshops and
-    training must be similarly recorded in a shared central location
-    (e.g., a common database).  All organizations with access to that
-    information must respect best practices for personal privacy (e.g,
-    may not contact workshop participants about other activities
-    without their explicit prior consent), and are expected to
-    coordinate their use of that information with other organizational
-    members of the SCF through the SCF Steering Committee and
-    Executive Director.
+    organization prefers.
+
+3.  All conversations must be archived in a shared central location
+    approved by the Steering Committee.
+
+### 3.4) Administration Fees for Internal Workshops Run by Member Organizations
+
+There is no administration fee for internal workshops run by member
+organizations.  (This is one of the benefits of becoming a member
+organization.)
+
+### 3.5) Administration Fees for Arranged Workshops for Not-For-Profit Organizations
+
+Arranged workshops for not-for-profit organizations such as
+universities and government institutions are charged:
+
+a.  US$1250 for the first workshop in any year;
+
+b.  an additional US$750/workshop for the second and subsequent
+    workshops in a year.
+
+### 3.6) Administration Fees for Arranged Workshops for For-Profit organizations
+
+Arranged workshops for companies and other for-profit organizations
+are charged:
+
+a.  US$5000 for the first workshop in any year;
+
+b.  an additional US$3000/workshop for the second and subsequent
+    workshops in a year.
+
+### 3.7) Attendance Fees for Workshops
+
+Host may charge learners a small fee for participating in a workshop.
+(In fact we encourage them to do so, since it significantly reduces
+the no-show rate.)  If they do this, they may use that money to pay
+part or all of any administration fee owed to the SCF.
+
+### 3.8) Fee Waivers
+
+The Steering Committee may reduce or waive workshop fees at its
+discretion.  In particular, Software Carpentry strives to be a global
+project and support a diverse set of organizations; any potential host
+that wishes to offer a workshop that would aid these goals is urged to
+contact the SCF to discuss waivers and other arrangements.
+
+### 3.9) SCF Use of Fees from For-Profit Organizations
+
+The SCF may use one quarter of administration fee revenue from
+for-profit organizations for its own purposes.  It must use the
+remaining three quarters of this money to underwrite the costs of
+workshops for not-for-profit organizations, such as their
+administration fees and/or instructors' travel and accommodation.
+
```

Comment:
I would move 3.9 up to directly after 3.6, and add a note that this is a pilot program and that terms may change. This goes double since (if I have understood the rest of the Fees committee correctly) we are thinking of doing discounts/waivers for companies too in the same vein as we may do for non-profits.